BridgeSDK/BridgeAPI/SBBActivityManager.m

```diff
@@ -103,11 +104,11 @@ - (NSArray *)filterTasks:(NSArray *)tasks forDaysAhead:(NSInteger)daysAhead andD
     NSArray *filtered = [tasks filteredArrayUsingPredicate:predicate];
     if (excludeValid) {
         // "valid things" are defined as the subset of the above things that the server would return.
-        // valid things expire after the start of today and are not marked as finished. the flag is
+        // valid things expire after now and are not marked as finished. the flag is
         // set so we want to exclude those things.
-        predicate = [NSPredicate predicateWithFormat:@"NOT ((%K == nil OR %K > %@) AND %K == nil)",
+        predicate = [NSPredicate predicateWithFormat:@"NOT ((%K == nil OR %K >= %@) AND %K == nil)",
```

Comment:
not sure whether '>' or '>=' is more correct in terms of matching server behavior, but since the odds of this code using precisely the same time instant the server used for checking against are infinitesimal, it shouldn't really matter either way.